lib/beaker/host/unix.rb

```diff
@@ -61,20 +61,24 @@ def self.foss_defaults
     end
 
     def self.aio_defaults
-      h = self.foss_defaults
-      h['puppetserver-confdir'] = '/etc/puppetlabs/puppetserver/conf.d'
-      h['puppetservice']  = 'puppetserver'
-      h['puppetbindir']   = '/opt/puppetlabs/agent/bin'
-      h['puppetbin']      = "#{h['puppetbindir']}/puppet"
-      h['puppetpath']     = '/etc/puppetlabs/agent'
-      h['puppetconfdir']  = "#{h['puppetpath']}/config"
-      h['puppetcodedir']  = "#{h['puppetpath']}/code"
-      h['puppetvardir']   = '/opt/puppetlabs/agent/cache'
-      h['distmoduledir']  = "#{h['puppetcodedir']}/modules"
-      h['sitemoduledir']  = '/opt/puppetlabs/agent/modules'
-      h['hieraconf']      = "#{h['puppetcodedir']}/hiera.yaml"
-      h['hieradatadir']   = "#{h['puppetcodedir']}/hieradata"
-      h
+      h = Beaker::Options::OptionsHash.new
+      h.merge({
+        'user'                  => 'root',
+        'group'                 => 'puppet',
+        'puppetserver-confdir'  => '/etc/puppetlabs/puppetserver/conf.d',
+        'puppetservice'         => 'puppetserver',
```

Comment:
Same note about what does this have to do with QENG-1841 / pathing changes?  At the very least, it should probably be in it's own commit / there should be something in the commit message indicating why these changes are being mixed in.